docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -0,0 +1,331 @@
+# Introduction
+
+In the following, we will give a basic classification of timeouts, explain some
+candidate causes for timeouts, and show ways to sometimes prevent them. We give
+a glimpse into the more theoretical background of timeouts in program
+verification in [this section](timeouts-theory.md)
+
+We classify Certora Prover timeouts as follows:
+1.  Timeouts that happen before SMT solvers are running 
+2.  Timeouts where the SMT queries in sum lead to a global timeout
+3.  Timeouts where a single SMT query could not be solved 
+
+Types 1. and 2. are signified by a hard stop of the Prover. That means the
+Prover ran until the global timeout (set via `--globalTimeout`, typically 2
+hours) and was forcefully shut down from everything it was doing. A message like
+"hard stop reached" appears in the "Global problems" pane of the report, and
+error symbols next to one or many rules.
+
+% made CERT-3797 so we get a clear indication to the user, hopefully
+% then, CVT should indicate via report-logs
+%  - whether there was a hard stop
+%  - whether SMT has started
+
+
+Type 3. is signified by a soft stop. This means an SMT solver shut down due to
+hitting the limit for a single SMT run (set via `--smt_timeout`). When running
+with default settings this means that we give up for the individual rule since
+in order to obtain a proof of correctness we need to solve every subproblem we
+generate (see documentation on [control-flow splitting](control-flow-splitting)
+for more details). 
+
+In the remainder we will focus on the mitigation of SMT timeouts, i.e., types 2.
+and 3. Non-SMT Timeouts (Type 1.) should be reported to Certora. 
+
+(timeout_causes)=
+# What Causes Timeouts?
+
+As a first step towards resolving a timeout, we need to diagnose its root
+causes. In our experience so far, the following are some of the most common
+reasons for SMT timeouts:
+
+ - Non-trivial amount of nonlinear arithmetic
```

Comment:
The `.` again. I see now there are also several cases below. So, I will not comment on that anymore.

Reply:
changing case ✔️ looking for others ..